lib/active_admin/error.rb

```diff
@@ -45,8 +45,6 @@ def self.capture
       raise new exception
     end
 
-    private
```

Comment:
Interesting. I think the intention here was to make the `database_error_classes` method private. Should we add `private_class_method :database_error_classes` here to make that happen?